game-core/src/main/java/games/strategy/util/PointFileReaderWriter.java

```diff
@@ -31,6 +31,11 @@
  * String -> a list of points, or string-> list of polygons.
  */
 public final class PointFileReaderWriter {
+
+  private static final Pattern pointPattern = Pattern.compile("\\s*\\(\\s*(\\d+)\\s*,\\s*(\\d+)\\s*\\)");
```

Comment:
If we have the knowledge, some examples of what these would match would be excellent to document.